### ISSUE TITLE
Protected Avatar in Public Folder 

### ISSUE THREAD

**Issue by Nordwig** [Neos-Metaverse/NeosPublic#2098]
## Is your tweak request related to a problem? Please describe.
When a user directly equips ( yellow lightning bolt ) an protectet Avatar from a public folder. ( Protection from another user )
The current equiped avatar gets deleted and the new cant be equiped.
So the user is left behind with no avatar and is not able to speak, interactions are still working.

## Describe what would you like tweaked
The "equip Avatar" yellow lightning bolt shouldnt be active when the selected Avatar is protectet by another user.

## Describe alternatives you've considered
1.) If a user without an equiped avatar trys to enter an avatar and is not allowed to, the default heads an Hands avatar should be equiped or the previus one should be restored.
2.) Check if the "equiping avatar" is alowed to be equiped by the User before destroying the current one.

## Additional context
Temporary Publicfolder for Testing: neosrec:///U-Nordwick/R-a9581740-06df-4b6e-b8c4-cb38a86062ad

**Comment by shadowpanther** [comment]
Shouldn't at least the default hmd-and-hands avatar equip instead if the selected one can't be equipped?
That, or check whether the avatar could be equipped first, before deleting the current avatar.

**Comment by TehTurk** [comment]
I mean be design you don't always need to have an avatar equipped. This might just be better off something waiting until new Inventory.

**Comment by shadowpanther** [comment]
I mean, having your avatar deleted and having no head to speak out of is not the most pleasant experience and shouldn't be triggered by trying to equip a random avatar from a public folder.

**Comment by TehTurk** [comment]
I agree, but then that's just letting the owner know in some form too. While I understand the concern this is also a mix of user error too. Generally you shouldn't be saving anything with AvatarProtection to a public folder. I'll mark this as low prio for now and we can mark it as a problem to consider when Inventory is touched

**Comment by shadowpanther** [comment]
That's one user's error/intention that is affecting other users. This might not be related to the inventory rework at all if the equipping mechanism itself could be changed to avoid this behavior or have a fallback default avatar in case of such an error.

**Comment by TehTurk** [comment]
I get it.

For now send me the users folder or dm me details on discord and I can run them down and speak to them for now?

**Comment by shiftyscales** [comment]
This has an underlying connection with other issues such as #1964.

This issue isn't really too relevant to the inventory rework @TehTurk, more so the licensing system I feel, e.g. attempting to equip a protected/licensed avatar by any means could prompt the user to purchase/provide a license for it, or if cancelled, wouldn't proceed in switching their avatar.

Technically this is everything working as intended from the perspective of SimpleAvatarProtection, it's preventing unauthorized users from wearing the equipped avatar, even when directly loaded.

What should be looked at more is how to handle cases where a user has no avatar equipped, and deal with that instead if their avatar has been removed by any means without a replacement.

**Comment by TehTurk** [comment]
Definitely a in both ballpark issue, my initial assessment was inventory for now because it had to deal with interactions and such and I figured Froox doesn't touch anything with old menus atm.

Hmm I don't know about the later, while from a UI standpoint that makes sense, but if you remove the default functionality as a failsafe mechanism you can run into some weirdness with logix nodes.

**Comment by shadowpanther** [comment]
Equip Avatar already checks for avatar protection and doesn't proceed to equip the avatar, so it wouldn't be affected.
Also, it's not about removing the functionality, but rather about adding it - e.g. having a fallback headset-and-hands avatar if your avatar is deleted for some reason.
